server/routes/createChain.ts

```diff
@@ -0,0 +1,129 @@
+import { Request, Response, NextFunction } from 'express';
+import Web3 from 'web3';
+import { slugify, urlHasValidHTTPPrefix } from '../../shared/utils';
+import { DB } from '../database';
+
+import { factory, formatFilename } from '../../shared/logging';
+const log = factory.getLogger(formatFilename(__filename));
+
+export const Errors = {
+  NoName: 'Must provide name',
+  InvalidNameLength: 'Name should not exceed 255',
+  NoSymbol: 'Must provide symbol',
+  InvalidSymbolLength: 'Symbol should not exceed 255',
```

Comment:
Do we want 255 to be the actual max char length for symbols? [Per this StackExchange comment](https://ethereum.stackexchange.com/a/25655), seems like there are no tokens with >10 characters, and almost all of them are <5.

Reply:
resolved